pycuda/gpuarray.py

```diff
@@ -1138,7 +1138,7 @@ def imag(self):
         else:
             return zeros_like(self)
 
-    def conj(self):
+    def conj(self, out=None):
```

Comment:
Could you add this to the documentation? (including a `.. versionchanged:: 2020.1.1` directive)